### PR TITLE
:pencil2: Fix some typos in French.

### PR DESCRIPTION
**Pull request by Zenigata**
Open for discussions about the typos I want to fix.

### CODE REVIEW

File: lang/fr/index.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ Introduction
 ------------
 
 Dans le monde de la gestion des logiciels, il existe un endroit redouté appelé
-"l'enfer des dépendances" (de l'anglais "dependency hell"). Plus votre système se
+« l'enfer des dépendances » (de l'anglais _dependency hell_). Plus votre système se
```
Comment (klehelley) [inline lang/fr/index.md]: It looks like that the spaces around the _guillemets_ are both standard ones. Ideally, the ones on the inside should be non-breaking spaces. This comment obviously also applies to the others _guillemets_ of the document. Ignore this remark if Jekyll transforms non-breaking spaces to standard spaces when generating the HTML page.

**Review by klehelley — APPROVED** [review]
This pull request looks OK to me. It fixes a lot of small errors here and there as well as updates typography to respect the French conventions better. One small caveat would be the lack of non-breaking spaces, but even without them the changes are welcome improvements.

File: lang/fr/index.md (hunk 1 above)
Comment (Zenigata) [reply lang/fr/index.md]: I used to have a blog using Jekyll and I usuallly typed the non breaking spaces. I can add them here or save this enhancement for another review. As you wish (and when I have another free time).

Comment (Zenigata) [reply lang/fr/index.md]: I have updated the changes with non-breaking spaces. Is it good for you? Thx!

Comment (klehelley) [reply lang/fr/index.md]: Perfect! Cheers. :)

**Comment by alexandrtovmach** [comment]
@Zenigata Thanks for contribution 👍 
@klehelley Thank you for review 🎉

**Pull request merged**